pkg/apis/velero/v1/pod_volume_restore_type.go

```diff
@@ -30,12 +30,17 @@ type PodVolumeRestoreSpec struct {
 	Volume string `json:"volume"`
 
 	// BackupStorageLocation is the name of the backup storage location
-	// where the restic repository is stored.
+	// where the backup repository is stored.
 	BackupStorageLocation string `json:"backupStorageLocation"`
 
-	// RepoIdentifier is the restic repository identifier.
+	// RepoIdentifier is the backup repository identifier.
 	RepoIdentifier string `json:"repoIdentifier"`
 
+	// UploaderType is the type of the uploader to handle the data transfer.
+	// +kubebuilder:validation:Enum=kopia;restic;""
+	// +optional
+	UploaderType string `json:"uploader-type"`
```

Comment:
Seems the convention is to use `camelCase` in the `json` tag

Reply:
I don't disagree, but it is done following the design.  
@Lyndon-Li I also think we should use camelCase here, if it sounds good to you, I'll make the change and also update the design in this PR.

Let me know your thoughts

Reply:
@reasonerjt It sounds good to me. Please help to update the design. Thanks.